lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1002,15 +1004,15 @@ def clear(self):
     def _button_press(self, event):
         if event.inaxes == self:
             self.button_pressed = event.button
-            self.sx, self.sy = event.xdata, event.ydata
+            self._sx, self._sy = event.xdata, event.ydata
             toolbar = getattr(self.figure.canvas, "toolbar")
             if toolbar and toolbar._nav_stack() is None:
                 self.figure.canvas.toolbar.push_current()
 
     def _button_release(self, event):
         self.button_pressed = None
         toolbar = getattr(self.figure.canvas, "toolbar")
-        if toolbar:
+        if toolbar and toolbar.mode not in (_Mode.PAN, _Mode.ZOOM):
```

Comment:
Might be useful to put a comment here about why we don't want to include PAN/ZOOM modes.

In `_button_press` above, it checks for `toolbar._nav_stack() is None`, do we need to update this to that check, or add the PAN/ZOOM mode check into the above function as well?

Yet another option may be to use what you're doing below in `_get_view()`
```python
if self.get_navigate_mode() is None:
```

Reply:
I like the `navigate_mode` solution. We do not need to add that check to `_button_press`, since that check is there to save the initial view before any transformations are applied.